src/main/java/org/opentripplanner/gtfs/RepairStopTimesForEachTripOperation.java

```diff
@@ -179,30 +162,19 @@ private void filterStopTimes(List<StopTime> stopTimes) {
             int dwellTime = st0.getDepartureTime() - st0.getArrivalTime();
             if (dwellTime < 0) {
                 issueStore.add(new NegativeDwellTime(st0));
-                if (st0.getArrivalTime() > 23 * SECONDS_IN_HOUR
-                        && st0.getDepartureTime() < 1 * SECONDS_IN_HOUR) {
-                    midnightCrossed = true;
-                    st0.setDepartureTime(st0.getDepartureTime() + 24 * SECONDS_IN_HOUR);
-                } else {
-                    st0.setDepartureTime(st0.getArrivalTime());
-                }
+                return false;
```

Comment:
Should we log to issue store here?

Reply:
It is done already on the line above the changes